seeds/007-sessions.js

```diff
@@ -0,0 +1,32 @@
+const dbName = 'sessions'
```

Comment:
is this just sample data? is it exported? does it make sense to have this defined here, versus elsewhere?

Reply:
It's just sample seed data, not meant for production